website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Enhancement: Edit in primary git branch in IDE"
+description: "February 2024: Edit access to the primary git branch while in the IDE. This was previously a read-only feature."
```

Comment:
i think we can clarify that they can commit to a new branch, just so it's super clear they can't commit changes in their main branch.

```suggestion
description: "February 2024: Edit directly on your primary git branch and commit those changes to a new branch when ready. This was previously a read-only feature." 
```

Reply:
wdyt?

Reply:
i want to make sure the description is super clear when it's announced in the slack community
<img width="920" alt="Screenshot 2024-02-23 at 15 17 31" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/4aaf2c51-6f4b-4a8a-b39a-4e8e949429ea">